### PR TITLE
XW-201 | Create SpotifyTag extension

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

File: extensions/wikia/SpotifyTag/SpotifyTagContoller.class.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	public static function onParserFirstCallInit( Parser $parser ) {
+		$parser->setHook( self::TAG_NAME, [ new static(), 'renderTag' ] );
+		return true;
+	}
+
+	public function renderTag( $input, array $args, Parser $parser, PPFrame $frame ) {
+		$isValid = $this->validator->validateAttributes( $args );
+
+		if ( !$isValid ) {
+			return '<strong class="error">' . wfMessage( 'spotify-could-not-render' )->parse() . '</strong>';
```
Comment (bognix) [inline extensions/wikia/SpotifyTag/SpotifyTagContoller.class.php]: spotify-tag to be consistant with remaining tags

**Comment by bognix** [comment]
LGTM

File: extensions/wikia/SpotifyTag/SpotifyTag.i18n.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+$messages = [];
+
+$messages['en'] = [
+	'spotify-tag-desc' => 'Spotify widget integration for the Wikia platform',
+	'spotify-tag-could-not-render' => 'Failed to render the Spotify widget. Please check if all required parameters are in place.',
+];
+
+$messages['qqq'] = [
+	'spotify-tag-desc' => '{{desc}}',
+	'spotify-tag-could-not-render' => 'Error message shown when the Spotify widget could not be loaded. Only happens if Spotify has a problem, or the user disallowed 3rd-party resources.',
```
Comment (rogatty) [inline extensions/wikia/SpotifyTag/SpotifyTag.i18n.php]: :no_good:

**Pull request merged**